spec.bs

```diff
@@ -1260,8 +1248,35 @@ an [=ordered map=] whose [=map/keys=] are [=strings=] and whose [=map/values=] a
     1. Set |global|'s
       [=InterestGroupBiddingScriptRunnerGlobalScope/is component auction=] to |isComponentAuction|.
     1. Set |global|'s [=InterestGroupBiddingScriptRunnerGlobalScope/interest group=] to |ig|.
-    1. Return the result of [=evaluating a script=] with |global|, |script|, "`generateBid`",
+    1. Let |startTime| be the current time.
+    1. Let |result| be the result of [=evaluating a script=] with |global|, |script|, "`generateBid`",
       |arguments|, and |timeout|.
+    1. Let |duration| be the current time minus |startTime| in milliseconds.
+    1. If |global|'s [=InterestGroupBiddingScriptRunnerGlobalScope/priority=] is not null:
+      1. Set |ig|'s [=interest group/priority=] to |global|'s
+        [=InterestGroupBiddingScriptRunnerGlobalScope/priority=].
+      1. Store |ig| in the browser’s [=interest group set=].
+    1. If |global|'s [=InterestGroupBiddingScriptRunnerGlobalScope/priority signals=] is not
+      [=map/is empty|empty=]:

```

Comment:
You can use

```suggestion
    1. If |global|'s [=InterestGroupBiddingScriptRunnerGlobalScope/priority signals=]
      [=map/is not empty=]:
```

You can tell because the [`<dfn>`](https://infra.spec.whatwg.org/#map-is-empty) includes `data-lt="is empty|is not empty"`.